src/index.d.ts

```diff
@@ -1249,26 +1229,10 @@ declare namespace admin.machineLearning {
     /** The size of the model. */
     readonly sizeBytes: number;
 
+    // At most one of these two will be specified.
```

Comment:
I think this should be stated in the type description so it gets included in the typedocs.

Reply:
done (Also, Kevin will review/fix the docs before release, once we get this submitted)

Reply:
In that case lets remove this line for now.

Reply:
done